src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Utilities/RazorProjectInfoFactory.cs

```diff
@@ -126,7 +126,7 @@ private static RazorConfiguration ComputeRazorConfigurationOptions(AnalyzerConfi
             configurationName,
             Extensions: [],
             UseConsolidatedMvcViews: true,
-            suppressAddComponentParameter);
+            SuppressAddComponentParameter: suppressAddComponentParameter);
```

Comment:
Why did you add a named argument here? It's unneeded and matches the name of the local.